app/views/hub/metrics/index.html.erb

```diff
@@ -21,79 +21,172 @@
     </div>
   </div>
 
-  <div>
+  <div class="scrollable-table-wrapper">
     <table class="index-table org-metrics-table">
       <thead class="index-table__head">
         <tr class="index-table__row">
           <th scope="col" class="index-table__header sortable" id="organization-name" width="40%">
             Organization Name
           </th>
           <th scope="col" class="index-table__header sortable" id="needs-attention-breaches">
-            Needs attention breaches<sup>1</sup>
+            Attention breaches<sup>1</sup>
```

Comment:
I shortened the names on these to try to mitigate the need to horizontally scroll.

Reply:
good call